==========================================
SLURM_JOB_ID = 332814
SLURM_NODELIST = gnode36
SLURM_JOB_GPUS = 1,2
==========================================
=> creating model 'resnet'
True
the number of model parameters: 2073556
Traceback (most recent call last):
  File "train.py", line 426, in <module>
    main()
  File "train.py", line 201, in main
    train_loss = train(train_loader, model, criterion, optimizer, epoch)
  File "train.py", line 260, in train
    output = model(input)
  File "/home/sandeep.nagar/anaconda3/lib/python3.7/site-packages/torch/nn/modules/module.py", line 727, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/home/sandeep.nagar/anaconda3/lib/python3.7/site-packages/torch/nn/parallel/data_parallel.py", line 161, in forward
    outputs = self.parallel_apply(replicas, inputs, kwargs)
  File "/home/sandeep.nagar/anaconda3/lib/python3.7/site-packages/torch/nn/parallel/data_parallel.py", line 171, in parallel_apply
    return parallel_apply(replicas, inputs, kwargs, self.device_ids[:len(replicas)])
  File "/home/sandeep.nagar/anaconda3/lib/python3.7/site-packages/torch/nn/parallel/parallel_apply.py", line 86, in parallel_apply
    output.reraise()
  File "/home/sandeep.nagar/anaconda3/lib/python3.7/site-packages/torch/_utils.py", line 428, in reraise
    raise self.exc_type(msg)
RuntimeError: Caught RuntimeError in replica 0 on device 0.
Original Traceback (most recent call last):
  File "/home/sandeep.nagar/anaconda3/lib/python3.7/site-packages/torch/nn/parallel/parallel_apply.py", line 61, in _worker
    output = module(*input, **kwargs)
  File "/home/sandeep.nagar/anaconda3/lib/python3.7/site-packages/torch/nn/modules/module.py", line 727, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/home/sandeep.nagar/seeds_project/CutMix-PyTorch-master/resnet.py", line 160, in forward
    x = self.fc(x)
  File "/home/sandeep.nagar/anaconda3/lib/python3.7/site-packages/torch/nn/modules/module.py", line 727, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/home/sandeep.nagar/anaconda3/lib/python3.7/site-packages/torch/nn/modules/linear.py", line 93, in forward
    return F.linear(input, self.weight, self.bias)
  File "/home/sandeep.nagar/anaconda3/lib/python3.7/site-packages/torch/nn/functional.py", line 1690, in linear
    ret = torch.addmm(bias, input, weight.t())
RuntimeError: mat1 dim 1 must match mat2 dim 0

